doc/source/container_builder.rst

```diff
@@ -0,0 +1,276 @@
+Building in a self-contained Environment
+========================================
+
+.. topic:: Abstract
+
+   This document describes how to build with KIWI using
+   dice as a container buildsystem in order to eliminate
+   any build host system requirements except the ones
+   dice pulls in
+
+Requirements
+------------
+
+The following components are needed in order to build in a
+self-contained environment:
+
+* :command:`dice` - a containment build system for KIWI.
+
+* :command:`docker` - a container framework based on the Linux
+  container support in the kernel.
+
+* docker build container for KIWI.
+
+* optionally :command:`vagrant` - a framework to run provision
+  and control virtual machines and container instances. vagrant
+  has a very nice way to provision a machine prior to running the
+  actual build. vagrant also supports docker as a provider which
+  makes it a perfect fit for complex provisioning tasks in
+  combination with the docker container system.
+
+Installing and Setting up dice
+------------------------------
+
+dice packages are available at the respective buildservice project
```

Comment:
I would rephrase that to:

The dice packages and sources are available at the following locations:

* Build service project: http://download.opensuse.org/repositories/Virtualization:/Appliances
* Sources: https://github.com/schaefi/dice